roles/edpm_download_cache/tasks/container_images.yml

```diff
@@ -9,6 +9,7 @@
     - download_cache
 
 - name: Download images for edpm_iscsid role
+  when: '"nova-storage" in edpm_services'
```

Comment:
We should also take into account the `edpm_services_override`,
maybe we can have a variable that defaults to `edpm_services` and gets `edpm_services_override` value if it is defined

Reply:
@slagle FYI

Reply:
I think we can have something like:
```yaml
edpm_running_services: {{ edpm_services_override | default(edpm_services) }}
```
so if you want the "default" services you can get `edpm_services` and if you want what is being executed you can get `edpm_running_services`

Reply:
That would be a good idea. The overrides are meant to take priority after all.

Reply:
It makes sense, I've added `edpm_download_cache_running_services` variable.